contributing.md

```diff
@@ -0,0 +1,141 @@
+
+*Ready*  
+An issue is ready if it’s possible to start working on it, i.e. there are no blocking dependencies and requirements are clear enough to start working. Dependencies are expressed through simple referencing of the blocking issue (e.g. dependes on #111), see details on referencing. Prioritized items (like bugs) are moved to the top of the ready column.
+The assignee must verify the readiness with the [Definition of Ready](#definition-of-ready).
+
+*Working*  
+When we start working on an issue, we move it to the working column. Ideally, every person should only work on one issue at a time. That way the working column provides an overview of who is currently working on what. Issues are only moved into or out of the working column by the person who is assigned. Issues in working are only reassigned by the person who is currently assigned.
+For every issue we open a feature branch that contains the corresponding issue number and additional info for convenience (using camelCaseFormatting, e.g. 111-featureDesciption).
+We include references to the corresponding issue in the commit messages.
+If the assignee thinks the issue is ready for review he/she moves it to review column and assigns a user for functional review.
+We add instructions and links for testing the changed behavior on the test system in the issue.
+
+*Review*  
+There are two kinds of Review:  
+a) Functional Review  
+If the reviewers find problems or have comments during the review, they describe the issues, providing links that show the behavior, and reassign the team member that submitted the issue, leaving the issue in the review column.
+If everything works as expected, the reviewers post a +1 comment on the issue and unassign themselves as reviewer.
+
+b) Code Review  
+To start a code review the person who worked on the issue assigns the corresponding pull request for code review.The pull request should be linked to the issue.
+Changes during the review process are created in additional commits which are pushed to the feature branch. They are added to the existing pull request automatically.
+At the end of the code review, the reviewer approves the pull request and reassigns the pull request to its original creator.
+
+*Done*  
+The creator of the pull request merges the pull request after checking the [definiton of done](#definition-of-done). After the merge the issue and the linked pull request move automatically to the done column. We delete feature branches after merging.
+
+### Definition of Ready
+The Definition of Ready describes a list of criteria which tickets have to meet to move from column 'Backlog' to 'Ready':
+
+- The person who implements the ticket is assigned and has every information to work on this ticket. Only the assignee can move the ticket to Ready.
+- The implementation of the ticket is planned for the next two (?) weeks and all participants have sufficient resources for the ticket.
+- choose reviewers already in grooming?
+
```

Comment:
I don't like to be assigned as a reviewer in "grooming" - if that means the issue is not already ready to be reviewed. Because if you are assigned you get a notification and I like to know that I am now responsible for it. Postponing the issue because I would have to wait because it is not ready for me to work on may make me forget about my assignement (I would have to "pull" the information about the readiness for it instead of be pushed).

Reply:
In general, for PRs I think we could make better use of the Reviewer/Assignee separation.

The way I understood the `choose reviewers already in grooming?` suggestion is not that they would necessarily be assigned during grooming, but that we could agree who would be a good reviewer for the issue. We could e.g. add that to a comment, like "will be reviewed by ...", and then assign only when it's ready for review.